buildenv/jenkins/getDependency

```diff
@@ -0,0 +1,38 @@
+#!groovy
+
+LABEL='ci.role.test&&hw.arch.x86&&sw.os.linux'
+
+stage('Queue') {
+    node("$LABEL") {
+       cleanWs()
+       testBuild()
+    }
+}
+
+def testBuild() {
+	def time_limit = 8
+	if(params.TIME_LIMIT) {
+		time_limit = params.TIME_LIMIT.toInteger()
+	}
+	timeout(time: time_limit, unit: 'HOURS') {
+		try {
+			if( params.BUILD_TYPE == "systemtest" ){
+			    sh 'curl -OLJks  "https://api.adoptopenjdk.net/v2/binary/nightly/openjdk8?openjdk_impl=hotspot&os=linux&arch=x64&release=latest&type=jdk"'
+                sh 'mkdir ${WORKSPACE}/j2sdk-image'
+                sh 'tar -xzf OpenJDK8U-jdk_x64_linux_hotspot*.gz -C ${WORKSPACE}/j2sdk-image --strip-components 1'
```

Comment:
What's the purpose of this tar.gz?

Reply:
We need the SDK to compile the checked-out mauve source code.